types/critters-webpack-plugin/index.d.ts

```diff
@@ -1,8 +1,13 @@
-import { Plugin } from "webpack";
+import { Compiler, WebpackPluginInstance } from "webpack";
 
 export default Critters;
-declare class Critters extends Plugin {
+declare class Critters implements WebpackPluginInstance {
     constructor(options?: Critters.CrittersOptions);
+    apply: (compiler: Compiler) => void;
+    getCssAsset(href: any, style: any): Promise<any>;
```

Comment:
Based on https://github.com/GoogleChromeLabs/critters/blob/main/packages/critters/src/index.js#L302

This method only accepts one argument of type `string`